lib/_emerge/actions.py

```diff
@@ -2095,7 +2095,7 @@ def action_info(settings, trees, myopts, myfiles):
                 if not atom.blocker:
                     atoms.append((x, atom))
 
-    myvars = sorted(set(atoms))
+    myvars = sorted(set(atoms), key=lambda t: str(t[0]))
```

Comment:
It looks like we can omit this sort because myvars is only used to populate the cp_map variable.